repos/system_upgrade/common/actors/udev/udevadminfo/tests/test_udevadminfo.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+from leapp.libraries.actor import udevadminfo
+from leapp.libraries.common import testutils
+from leapp.libraries.stdlib import api, run
+from leapp.models import UdevAdmInfoData
+
+CUR_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+def test_udevadminfo(monkeypatch):
+
+    with open(os.path.join(CUR_DIR, 'files', 'udevadm_database'), 'r') as fp:
```

Comment:
Nit: Now that `udevadm`  correct execution is checked you could also add a unit test that StopActorExecution is raised.

Reply:
Good Idea, thank you